lib/beaker-benchmark/helpers.rb

```diff
@@ -26,6 +28,8 @@ module Helpers
         PROC_DISK_READ_INDEX = 11
         PROC_DISK_WRITE_INDEX = 13
 
+        TMP_DIR = 'tmp/atop'
```

Comment:
did you mean for this to /tmp... ?

Reply:
No, I changed it to write to a directory under the working directory.